cmd/controller/main.go

```diff
@@ -89,6 +90,7 @@ func main() {
 		allocation.NewController(manager.GetClient(), clientSet.CoreV1(), cloudProvider),
 		termination.NewController(ctx, manager.GetClient(), clientSet.CoreV1(), cloudProvider),
 		node.NewController(manager.GetClient()),
+		&nodemetrics.Controller{KubeClient: manager.GetClient()},
```

Comment:
optional: Might want to make a constructor just to stay uniform with the others controllers.